### PR TITLE
Docker installation instructions in top-level docs

### PR DESCRIPTION
**Pull request by mswilkison**

### CODE REVIEW

**Comment by vepkenez** [comment]
Ideally it would be great if someone (besides me) could test the worker install instructions.

File: docs/source/guides/installation_guide.md [hunk 1]
```diff
@@ -103,6 +104,21 @@ Here is the recommended procedure for setting up `nucypher` in this fashion:
...
     ```
 
+### Standard Docker Install
+1. Install [Docker](https://docs.docker.com/install/)
+
+2. (Optional) Follow these post install instructions: https://docs.docker.com/install/linux/linux-postinstall/
+
+3. Get the latest nucypher image:
+
+  `(maybe sudo) docker pull nucypher/nucypher:latest`
```
Comment (michwill) [inline docs/source/guides/installation_guide.md]: Eh, I think you probably do it wrong if you need sudo?

File: docs/source/guides/installation_guide.md [hunk 2]
```diff
@@ -142,36 +158,70 @@ Install the Solidity compiler (solc):
...
+* Run the tests:
+`docker-compose run nucypher-dev pytest`
+* Start up an Ursula:
+`docker-compose run nucypher-dev nucypher ursula run --dev --federated-only`
+* Open a shell:
+`docker-compose run nucypher-dev bash`
+
+* try some of the scripts in `dev/docker/scripts/`
+
+**tested on (Ubuntu 16, MacOS 10.14, Windows 10)*
```
Comment (fjarri) [inline docs/source/guides/installation_guide.md]: Are the asterisks supposed to be mismatched here?

File: docs/source/guides/installation_guide.md [hunk 3]
```diff
@@ -22,7 +23,7 @@ As of August 2019, we are working with Python 3.6, 3.7, and 3.8.
 
 ## Standard Installation
 
-We recommend installing `nucypher` with either `pip` or `pipenv`
+We recommend installing `nucypher` with either `pip` or `pipenv` or `docker`
```
Comment (cygnusv) [inline docs/source/guides/installation_guide.md]:
```suggestion
We recommend installing `nucypher` with either `pip`, `pipenv` or `docker`
```

File: docs/source/guides/installation_guide.md [hunk 4]
```diff
@@ -5,6 +5,7 @@
 
 * [System Requirements and Dependencies](#System-Requirements-and-Dependencies)
 * [Standard Installation](#Standard-Installation)
+* [Docker Installation](#Docker-Installation)
```
[the remark this answers is not in the document]
Comment (cygnusv) [reply docs/source/guides/installation_guide.md]: The link to dev installation is in L10.

File: docs/source/guides/installation_guide.md (hunk 3 above)
Comment (mswilkison) [reply docs/source/guides/installation_guide.md]: ✅

File: docs/source/guides/installation_guide.md (hunk 2 above)
Comment (mswilkison) [reply docs/source/guides/installation_guide.md]: ✅

File: docs/source/guides/installation_guide.md (hunk 1 above)
Comment (mswilkison) [reply docs/source/guides/installation_guide.md]: ✅

File: docs/source/guides/installation_guide.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (mswilkison) [reply docs/source/guides/installation_guide.md]: Made Docker Installation an h2 header so it shows up in the sidebar TOC

File: docs/source/guides/ursula_configuration_guide.rst [hunk 5]
```diff
@@ -74,9 +82,34 @@ If your installation is non-functional, be sure you have the latest version inst
...
+.. code:: bash
+
+    export NUCYPHER_KEYRING_PASSWORD=<your keyring password>
+    export MY_IP=$(wget -q -O - ifconfig.me);
+    export NUCYPHER_WORKER_ADDRESS=<eth account checksum of your worker>
+    export NUCYPHER_STAKER_ADDRESS=<eth account checksum of your staker>
+    export NUCYPHER_WORKER_ETH_PASSWORD=<your eth account password>
+
+    # init your worker
+    docker run -v /home/ubuntu:/root/.local/share/ -e NUCYPHER_KEYRING_PASSWORD -it nucypher/nucypher:latest nucypher ursula init --provider /root/.local/share/geth/.ethereum/goerli/geth.ipc --poa --worker-address $NUCYPHER_WORKER_ADDRESS --staker-address $NUCYPHER_STAKER_ADDRESS --rest-host $MY_IP --sync
```
Comment (derekpierre) [inline docs/source/guides/ursula_configuration_guide.rst]: **NOTE**: `--sync` is no longer a valid option for `nucypher ursula init` and `nucypher ursula run` (L107 below)

Comment (mswilkison) [reply docs/source/guides/ursula_configuration_guide.rst]: https://github.com/nucypher/nucypher/pull/1432

**Pull request merged**